components/layout_2020/flow/inline.rs

```diff
@@ -604,7 +604,7 @@ pub(super) struct InlineFormattingContextState<'a, 'b> {
     /// of the inline box is the state popped from the stack.
     inline_box_state_stack: Vec<InlineBoxContainerState>,
 
-    /// A vector of fragment that are laid out. This includes one [`Fragment::Anonymous`]
+    /// A vector of fragment that are laid out. This includes one Fragment::Anonymous
```

Comment:
we should not remove [&#96; and &#96;], we still want it to be a link. `Anonymous` was removed in this [PR](https://github.com/servo/servo/pull/31341) hence Rust doc is complaining,  this should be [&#96; Fragment::Positioning &#96;] now.